src/index.tsx

```diff
@@ -19,12 +19,13 @@ import authenticationManager from "./authentication/util/authenticationManager";
 import {createReduxStore} from "./core/redux/storeUtils";
 import {ReduxStoreShape} from "./core/redux/models/store";
 import authenticationApi from "./authentication/api/authenticationApi";
+import {generateAuthorizationHeaderValue} from "./core/network-manager/networkUtils";
 
 // initSentry();
 const networkManager = new NetworkManager({
   headers: {
     common: {
-      Authentication: authenticationManager.getToken()
+      Authorization: generateAuthorizationHeaderValue(authenticationManager.getToken())
```

Comment:
I'm not sure about this. Since the BE team changes the authentication type to session auth, probably we won't need this header any more in the new projects

Reply:
You may be right. However, currently, it is a bug. 
Also, as far as I know, the backend team still uses their starter package with token authentication. I think it's important to be in sync with them.

Reply:
Yes we can keep the token authentication for now and add a guideline for switching to cookie authentication